routes/index.ts

```diff
@@ -12,5 +13,6 @@ app.use('/health', health);
 app.use('/users', users);
 app.use('/calendar', calendar);
 app.use('/events', events);
+app.use('/timezone', timezone);
```

Comment:
why are we creating seperate route, can we not add to existing route?

Reply:
i think we can have it under users, since it comes under userOnboarding

Reply:
/timezone for now. Will review later if needed